src/App.css

```diff
@@ -13,20 +13,21 @@
   }
 }
 
+
+
 .App-header {
-  background-color: white; /* #282c34 */
```

Comment:
Colors can be importing from CSS like this https://css-tricks.com/getting-javascript-to-talk-to-css-and-sass/

Reply:
we are not using this class, this is default created with create-react-app.
I will remove this.
thanks for link